packages/marketplace/src/sagas/webhook-subscriptions.ts

```diff
@@ -22,10 +22,10 @@ export const fetchSubscriptions = async (applicationId: string) => {
   return response
 }
 
-export const fetchWebhookTopic = async () => {
+export const fetchWebhookTopic = async (applicationId: string) => {
```

Comment:
move this function to `services/webhooks.ts` file

Reply:
![image](https://user-images.githubusercontent.com/22660833/81288281-89361480-908e-11ea-8475-b66d2eddedb5.png)
Add more params like this to function for reusable